src/imageTools/probe.js

```diff
@@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
         return suv;
     }
     ///////// BEGIN IMAGE RENDERING ///////
+    function pointNearTool(data, coords) {
+        var point = {
+            left: data.handles.end.x,
+            top: data.handles.end.y,
+            width: 0,
+            height: 0
+        };
+
+        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
```

Comment:
Nope, it doesn't work.

What I did with my code was to treat the point like a rectangle.

distance function expects not complex objects, but numbers. The output 
is NaN in your case.

I will have a closer look this evening at the code, see if I can pull it 
through.

Sorry about the bad spaces, I'm kind-of new at social coding + my editor 
didn't saw the right spacing for the source files (probably because I'm 
on Windows).

Thanks for the help and suggestions.

On 19-May-2015 4:00 PM, Erik Ziegler wrote:
>
> In src/imageTools/probe.js 
> <https://github.com/chafey/cornerstoneTools/pull/41#discussion_r30595770>:
>
> > @@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
> >          return suv;
> >      }
> >      ///////// BEGIN IMAGE RENDERING ///////
> > +    function pointNearTool(data, coords) {
> > +        var point = {
> > +            left: data.handles.end.x,
> > +            top: data.handles.end.y,
> > +            width: 0,
> > +            height: 0
> > +        };
> > +
> > +        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
>
> This should probably be
>
> var  distanceToPoint=  cornerstoneMath.point.distance(point, coords);
>
> but you should test it to make sure it works.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/chafey/cornerstoneTools/pull/41/files#r30595770>.
>